crates/optimism/primitives/src/proof.rs

```diff
@@ -0,0 +1,335 @@
+use crate::{
+     Receipt, ReceiptWithBloom,B256,
+};
+
+/// Calculates the receipt root for a header.
+// #[cfg(feature = "optimism")]
+pub fn calculate_receipt_root_optimism(
```

Comment:
please ensure this is actually used where we currently use it (i.e. fix the import paths elsewhere), see e.g. https://github.com/search?q=repo%3Aparadigmxyz%2Freth%20calculate_receipt_root_optimism&type=code